platforms/linux-ppc64le/Dockerfile

```
@@ -34,6 +34,7 @@ RUN \
     --no-modify-path \
     --profile minimal \
     && \
+  rustup default nightly && \
   rustup target add powerpc64le-unknown-linux-gnu && \
```

```suggestion
    --profile minimal \
    --target powerpc64le-unknown-linux-gnu \
    --default-toolchain nightly \
    && \
```
(I also changed it to use `--target ...`  instead of `rustup target add ...`)